src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx

```diff
@@ -89,8 +89,8 @@ export default class ImageCard extends Component<IProps, IState> {
 
 		let jsx: string | React.ReactNode = <React.Fragment />;
 		switch (status) {
-			case 'error': jsx = errorJSX(); break;
-			case 'build_error': {
+			// case 'error': jsx = errorJSX(); break;
```

Comment:
why we need that commentary? remove pls

Reply:
Yes. Good code describes itself. We are using git.
All code history is saved, so we can be sure that any code, that you removing, can be restored lately if need.